toolkit/scripts/toolkit.mk

```diff
@@ -12,36 +12,90 @@ toolkit_component_extra_files = \
 	$(PROJECT_ROOT)/LICENSES-AND-NOTICES/LICENSE.md \
 	$(toolkit_root)/.gitignore
 
-mariner_repos_dir = $(SPECS_DIR)/mariner-repos
+mariner_repos_dir = $(PROJECT_ROOT)/SPECS/mariner-repos
```

Comment:
Never really gave it any thought, but should toolkit packaging be expected to work for non-standard layouts? (ie `SPECS_DIR=/path/to/my/SPECS`)

Reply:
The value of `SPECS_DIR` should have had no relevance to building the toolkit before the changes from this PR. It only needed the repo files and they are always in the same place, so this was just a corner case bug we never hit, because there was no point in setting `SPECS_DIR` to build the toolkit tarball.

With the current changes the snapshot relies on the value of `SPECS_DIR` but we still want to look for the repo files inside the core specs.